### PR TITLE
Draft Kubernetes Pilot Mentor Guide

### PR DESCRIPTION
**Pull request by chrislovecnm**
This is the initial draft of the K8s Pilot program mentor guide. The
intention of this document is to help mentors be successful. I included
a README that is a placeholder and needs some TLC.

/assign @sarahnovotny @parispittman 

## TODO

- [ ] Move this document to the correct folder if it is the wrong place
- [x] Work on the section about learning types
- [ ] Work on culture section
- [ ] Basic README

## Open Questions

Do we need a summary?

### CODE REVIEW

File: contributors/mentoring/draft-kubernetes-pilot-mentor-guide.md [hunk 1]
```diff
@@ -0,0 +1,370 @@
...
+      * [Visual](#visual)
+      * [Credit](#credit)
+   * [Community Culture Element to Impart](#community-culture-element-to-impart)
+   * [Time Expectations](#time-expectations)
+   * [When to ask for help](#when-to-ask-for-help)
+   * [Summary](#summary)
+
+## What is this document about
+
+This document is a guide for Mentors involved in the Kubernetes Pilot Program.
```
Comment (nikhiljindal) [inline contributors/mentoring/draft-kubernetes-pilot-mentor-guide.md]: Link to more details on the pilot program? How do ppl sign up, when does it start, who is eligible, etc

Comment (chrislovecnm) [reply contributors/mentoring/draft-kubernetes-pilot-mentor-guide.md]: Should be in the readme in the directory of this folder, yes we need that info, but should we have that in this doc as well?

Comment (nikhiljindal) [reply contributors/mentoring/draft-kubernetes-pilot-mentor-guide.md]: Sorry if i jumped the gun, i was curious. Will wait :)

Agree that there is no need to repeat it here. Linking to another doc that has that info should be sufficient.

Comment (chrislovecnm) [reply contributors/mentoring/draft-kubernetes-pilot-mentor-guide.md]: No worries! This is a alpha for this program, we are just figuring out everything ourselves. Ping me on slack if you have questions.

**Comment by chrislovecnm** [comment]
/hold

Need to refactor this some, break into smaller docs, and move them to the correct folder.

File: contributors/mentoring/draft-kubernetes-pilot-mentor-guide.md [hunk 2]
```diff
@@ -0,0 +1,370 @@
...
+
+> Mentoring is to support and encourage people to manage their own learning so
+that they may maximise their potential, develop their skills, improve their
+performance and become the person they want to be.
+
+- Eric Parsloe, The Oxford School of Coaching & Mentoring
+
+Mentoring is not coaching, but leading and encouraging people to reach their
+potential.  Mentee’s must be asked to take responsibility for their growth and
```
Comment (ConnorDoyle) [inline contributors/mentoring/draft-kubernetes-pilot-mentor-guide.md]: grammar nit: s/Mentee's/Mentees

File: contributors/mentoring/draft-kubernetes-pilot-mentor-guide.md [hunk 3]
```diff
@@ -0,0 +1,370 @@
...
+confidence and encourage.
+
+Anne Isabella Thackeray Ritchie in her novel, Mrs. Dymond talked about how to
+teach and mentor.  Rephasing her writing to the context of Mentoring; if we
+continuously answer a Mentee’s questions, and give them answers, we are limiting
+their growth.  If we teach them to find the answers and teach them how to
+contribute to Kubernetes, we help the Mentee become a lifetime Open Source
+Contributor.
+
+The Mentor’s role is to teach Mentees’ how to contribute to the Kubernetes
```
Comment (ConnorDoyle) [inline contributors/mentoring/draft-kubernetes-pilot-mentor-guide.md]: s/Mentees'/Mentees

File: contributors/mentoring/draft-kubernetes-pilot-mentor-guide.md [hunk 4]
```diff
@@ -0,0 +1,370 @@
...
+it is more critical to build productive contributors to our community.
+
+## Benefits of Being a Mentor
+
+Being mentor typically forces personal growth for not only the Mentee but also
+the Mentor.  Often a Mentor’s leadership and communications skills are improved.
+A Mentor is given back to the Kubernetes Community as a whole and helps improve
+the organization’s culture direction.  It often helps our perspective, as we can
+view the community through the eyes of a newcomer, which differs far from our
+current perspective.  Mentor learn about people; how they learn, how they
```
Comment (ConnorDoyle) [inline contributors/mentoring/draft-kubernetes-pilot-mentor-guide.md]: s/Mentor/Mentors

File: contributors/mentoring/draft-kubernetes-pilot-mentor-guide.md [hunk 5]
```diff
@@ -0,0 +1,370 @@
...
+
+### Communication
+
+How we act, carry ourselves, and approach people is critical as Mentors.
+Communication is a  key component of successful conduct as a mentor.
+Interrupting and not listening the people that we are communicating with are key
+factors that allow communication to fail.  Allow people to finish their
+statements, don’t interrupt.  Not interrupting may seem a simple habit to
+acquire, but again it is an art form.  If you have a habit of interrupting those
+you interact with, take a beat, start talking when a person is done with there
```
Comment (ConnorDoyle) [inline contributors/mentoring/draft-kubernetes-pilot-mentor-guide.md]: s/there/their

File: contributors/mentoring/draft-kubernetes-pilot-mentor-guide.md [hunk 6]
```diff
@@ -0,0 +1,370 @@
...
+communication.
+
+Once a person has completed what they have said, our job is then to understand
+what they said.  We are all different people, with diverse backgrounds, often
+from different countries, and cultures.  We may have heard the words that a
+person related, but 80% of communication is nonverbal, and the words of one
+person, may not the words of another.  Ask clarifying questions during the
```
Comment (ConnorDoyle) [inline contributors/mentoring/draft-kubernetes-pilot-mentor-guide.md]: s/may not/may not be

File: contributors/mentoring/draft-kubernetes-pilot-mentor-guide.md [hunk 7]
```diff
@@ -0,0 +1,370 @@
...
+put after inadequate self-criticism. But every question is a cry to understand
+the world. There is no such thing as a dumb question.
+
+― Carl Sagan, The Demon-Haunted World: Science as a Candle in the Dark
+
+There are no dumb questions. Allow people to feel that it is ok, to feel like
+you are a n00b.  We were not born with Kubernetes in our blood, we all started
+at zero.
+
+Consistency develops trust.  It may seem tripe, but do what you say you are
```
Comment (ConnorDoyle) [inline contributors/mentoring/draft-kubernetes-pilot-mentor-guide.md]: suggest: s/tripe/trite

File: contributors/mentoring/draft-kubernetes-pilot-mentor-guide.md [hunk 8]
```diff
@@ -0,0 +1,370 @@
...
+promise.  Under promise and over deliver. Setting Goals
+
+If you have any doubts that goal setting is critical see this Google
+[search](https://www.google.com/search?safe=on&q=why+set+goals).  We do not just
+set goals, but the goals must be written, with a timeline, measurable, and
+reachable.
+
+## Learning Styles
+
+There are three prodominate learning styles; Visual, Auditory, and Kinesthetic
```
Comment (ConnorDoyle) [inline contributors/mentoring/draft-kubernetes-pilot-mentor-guide.md]: s/prodominate/predominant

File: contributors/mentoring/draft-kubernetes-pilot-mentor-guide.md [hunk 9]
```diff
@@ -0,0 +1,370 @@
...
+Learning Styles. People tend to have a primary learning type and a secondary
+style. If a mentor and a mentee can identity there perfer learning styles, it
```
Comment (ConnorDoyle) [inline contributors/mentoring/draft-kubernetes-pilot-mentor-guide.md]: s/there prefer/their preferred

File: contributors/mentoring/draft-kubernetes-pilot-mentor-guide.md [hunk 10]
```diff
@@ -0,0 +1,370 @@
...
+### Auditory
+
+Auditory learners often talk to themselves. They also may move their lips and
+read out loud. They may have difficulty with reading and writing tasks. They
+often do better talking to a colleague or a tape recorder and hearing what was
+said. To integrate this style into the learning environment:
+
+- Begin new material with a brief explanation of what is coming. Conclude with a
+  summary of what has been covered. This is the old adage of "tell them what
+  they are going to lean, teach them, and tell them what they have learned."
```
Comment (ConnorDoyle) [inline contributors/mentoring/draft-kubernetes-pilot-mentor-guide.md]: s/lean/learn

**Comment by castrojo** [comment]
/sig contributor-experience

Hey Chris, how do you feel about discussing this broadly at a SIG contributor experience meeting? I feel like we should get more eyeballs on this as it's a large chunk of work.

**Comment by chrislovecnm** [comment]
@castrojo no problem. I do need to chunk it up more into separate docs. Paris’ said the feedback was it was a lot of words.

File: contributors/mentoring/draft-kubernetes-pilot-mentor-guide.md [hunk 11]
```diff
@@ -0,0 +1,370 @@
+---
+kep-number: draft-20172411
+title: My First KEP
```
Comment (mistyhacks) [inline contributors/mentoring/draft-kubernetes-pilot-mentor-guide.md]: What is a KEP?

Comment (xiangpengzhao) [reply contributors/mentoring/draft-kubernetes-pilot-mentor-guide.md]: FYI: https://github.com/kubernetes/community/blob/master/contributors/design-proposals/architecture/1-kubernetes-enhancement-proposal-process.md

Comment (chrislovecnm) [reply contributors/mentoring/draft-kubernetes-pilot-mentor-guide.md]: Typo ;)

**Comment by chrislovecnm** [comment]
Closing as we have another inbound

**Pull request closed without merging**